tests/unit/components/enhanced-table/enum-filter/EnumFilter.spec.jsx

```diff
@@ -0,0 +1,81 @@
+import { vi } from 'vitest'
+import { render, fireEvent, screen } from '@testing-library/react'
+
+import EnumFilter from '~/components/enhanced-table/enum-filter/EnumFilter'
+
+const mockedProps = {
+  column: {
+    filterEnum: [
+      {
+        label: 'adminTable.active',
+        value: 'active'
+      }
+    ]
+  },
+  filter: '',
+  setFilter: vi.fn(),
+  clearFilter: vi.fn()
+}
+
+describe('EnumFilter test', () => {
+
+  it('should open menu after clicking on filter icon', () => {
+    const { rerender } = render(<EnumFilter { ...mockedProps } />) 
+    rerender(<EnumFilter { ...mockedProps } />)

```

Comment:
why do you need rerender immediately after render?

Reply:
> why do you need rerender immediately after render?

fixed